checkers/testdata/octalLiteral/positive_tests.go

```diff
@@ -76,7 +75,12 @@ func (os *OpenServer) Init(x int) {
 	os.x = x
 }
 
-func warningsOs() {
+func warningsFs() {
+	/*! use new octal literal style, 0o555 */
+	_ = fs.FileMode(0555)
+}
+
+func warningsOsOpenFile() {
```

Comment:
Similar to `negative_tests.go`, this could be a single `warningsOs()` including both the `OpenFile()` and the `WriteFile()` calls.